Javilonte/nft-marketplace
# Template installation

## Changes to be added
- Add files from template (images, styles, JS files, etc)
- Refactor code for everything to work as expected
- Create a route to have all the pages from the template (`/template`)
- Create new pages from the corresponding template pages: `create.js`, `dashboard.js`, `assets.js`, `index.js`
- Tweak styles files for them to adapt to the project
- New info in the `README` about the project

## How to test
The testing consists just in verifying the styles for the pages and if there are the new pages from the template. These are the steps:

- Clone the my fork: `git clone git@github.com:ZambranoBeicker/nft-marketplace.git`
- Execute: `npm install` and follow installation instructions from the `README`
- Got to the next routes:
  -  `http://localhost:3000`
  -  `http://localhost:3000/create`
  -  `http://localhost:3000/dashboard`
  -  `http://localhost:3000/assets`
